containers/hawkular-openshift-security-filter/src/main/java/org/hawkular/openshift/auth/TokenAuthenticator.java

```diff
@@ -490,17 +487,18 @@ protected void stringDone(String body) {
                     boolean allowed = allowedNode != null && allowedNode.asBoolean();
                     onRequestResult(serverExchange, connection, allowed);
                 } catch (IOException e) {
-                    onRequestFailure(serverExchange, connection, e);
+                    onRequestFailure(serverExchange, connection, e, true);
                 }
             } else {
                 IOException e = new IOException(StatusCodes.getReason(clientExchange.getResponse().getResponseCode()));
-                onRequestFailure(serverExchange, connection, e);
+                // Retries will not help to solve the issue
+                onRequestFailure(serverExchange, connection, e, false);
```

Comment:
I tried to search the Kubernetes documentation on which return codes are available, but the documentation in this sense is useless. 503 is service unavailable, transforming it to 500 sounds like a proper choice in my opinion? It's not necessarily quickly available again.